bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/StaticImage.java

```diff
@@ -49,6 +49,16 @@ default String getValue() {
         return null;
     }
 
+    /**
+     * Retrieves the parsed img value to be displayed.
+     *
+     * @return the parsed img value to be displayed, or {@code null} if no value can be returned
+     * @since com.adobe.cq.forms.core.components.models.form 0.0.1;
+     */
+    default String getSource() {
```

Comment:
Why do you need this API now, can you remove it

Reply:
removed